rohmu/object_storage/base.py

```diff
@@ -82,6 +82,9 @@ def __init__(
         self.notifier = notifier or NullNotifier()
         self.stats = StatsClient(statsd_info)
 
+    def close(self) -> None:
```

Comment:
I do not understand who is calling this close call?
I could not find api in this repo which will enforce the `close` call.
Should we then add `close` call to the destructor to enforce at least postponed resource cleanup?

Reply:
Otherwise a client pool may be used. Or any other simple way which makes it easy to use and reliable

Reply:
Close is part of the API for this class.  A docstring might help here.

Reply:
Files, sockets, etc... need to be closed to reliably not have leaks, the same is true for APIs that use them.

I'll add a docstring to make it clear that this is part of the API.